src/EditorFeatures/Core/Shared/Preview/PreviewWorkspace.cs

```diff
@@ -60,6 +60,11 @@ public override void OpenDocument(DocumentId documentId, bool activate = true)
             this.OnDocumentOpened(documentId, text.Container);
         }
 
+        /// <summary>
+        /// Puts the specified additional document into the open state.
+        /// </summary>
+        /// <param name="documentId">The <see cref="DocumentId"/> to open.</param>
+        /// <param name="activate">Ignored - not necessary for additional documents.</param>
```

Comment:
Not sure I get this comment -- opening an additional document activated vs. not is a meaningful difference. It's that we don't care for the sake of the PreviewWorkspace, right?